fleetshard/pkg/central/operator/upgrade.go

```diff
@@ -4,42 +4,80 @@ package operator
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	"github.com/golang/glog"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
 	"github.com/stackrox/acs-fleet-manager/fleetshard/pkg/central/charts"
 	"helm.sh/helm/v3/pkg/chart"
 	"helm.sh/helm/v3/pkg/chartutil"
-	apiErrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
 	operatorNamespace = "stackrox-operator"
 	releaseName       = "rhacs-operator"
-	operatorImage     = "quay.io/rhacs-eng/stackrox-operator:3.74.0"
+	crdKind           = "CustomResourceDefinition"
+	// maximum number of characters for operator deployment suffix
+	// See rhacs-operator-deployment.yaml for more details
+	maxOperatorDeploymentSuffixLength = 29
 )
 
+func parseOperatorImages(images []string) ([]chartutil.Values, error) {
+	var operatorImages []chartutil.Values
+	uniqueTags := make(map[string]bool)
+	for _, img := range images {
+		if strings.Contains(img, ":") {
+			strs := strings.Split(img, ":")
+			if len(strs) != 2 {
+				return nil, fmt.Errorf("failed to split image and tag from %s", img)
+			}
+			tag := strs[1]
```

Comment:
```suggestion
			repo, tag := strs[0], strs[1]
```
Then using `repo` below instead of `strs[0]` is a bit neater.